### DISCUSSION TITLE
您好，我有一些关于“加载CSV数据” 的问题

### DISCUSSION THREAD

**Discussion by 8ShuDaZhaoFeng8** [CommandPost/CommandPost#3039] [Technical Support]
您好，我刚刚在使用CommandPost的过程中，发现了在“工具箱”的“FCPXML标题”里面有一个“加载CSV数据”我对这个功能十分感兴趣。

FCPX这款软件无法识别素材的时间码一直都是FCPX与达芬奇之间的沟壑

之前我所使用的方法是，先将所有素材拖进达芬奇，从达芬奇中导出元数据，会得到一份CSV数据，然后通过 Numbers 表格软件打开这份CSV文件，删除除开起始时间码以外的所有数据，然后将所有起始时间吗更改成00:00:00:00 ，再将这份CSV文件与达芬奇中的元数据合并，使得达芬奇中的元数据中的起始时间码也是00:00:00:00 ，再将FCPX中的XML文件导入达芬奇，需要链接的素材在媒体池中寻找，这样才能使得FCPX的时间线能够达芬奇中呈现出来，并且进行调色。

而我之所以激动，是因为在CommandPost 中发现“加载CSV数据”这一功能，于是我尝试在达芬奇中先导入所有的素材，并且直接导出一份后缀为“CSV” 的元数据文件，然后将这份文件，通过“加载CSV数据”这一功能希望能替换掉FCPX中素材的起始时间码，可能是误打误撞，我非常确信，我第一次尝试，成功了，并且我从FCPX中导出XML文件，直接可以通过达芬奇打开，中间没有任何错误，甚至通过达芬奇的离线参考片段 进行检查，也没有错误。

第一次可能素材少，且都在一个文件夹内的素材，针对大型项目以及素材管理规范的项目来说，参考的意义不大，只能说是nice try 。  于是我尝试以一个正常流畅，正常素材量进行测试，我发现，好像经过我重命名后的素材可以修改时间码，但是没有重命名的素材时间码修改不了。

于是我进行了第三次尝试，这次我只有三个文件夹，其中两个文件夹中都是SNOY素材，一个文件夹中是大疆的航拍素材，其中，有一个装有SNOY素材的文件夹中的所有素材，全部重新命名。其余两个文件夹没有进行重命名。

这次··一个都没有成功，前前后后尝试了3个小时，没有一次成功，这次无论是重命名素材、还是没有重命名的素材。我甚至尝试，将所有素材全部重命名，也没有成功。我希望能得到一些帮助，是我在这项功能的使用方式上存在错误，还是说我忽略了哪些操作。

希望能帮我答疑解惑，感谢

**Comment by latenitefilms** [comment]
**Translated:**

> Hello, in the process of using CommandPost, I found that there is a "Load CSV Data" in the "FCPXML Header" of the "Toolbox". I am very interested in this function.
> 
> FCPX this software does not recognize the timecode of the material has always been the chasm between FCPX and Da Vinci
> 
> The method I used before was to drag all the footage into DaVinci, export the metadata from DaVinci, get a CSV data, then open the CSV file with Numbers software, delete the start timecode All the data except the CSV file, then change all the start times to 00:00:00:00 , and then merge this CSV file with the metadata in DaVinci, so that the start time in the metadata in DaVinci The time code is also 00:00:00:00, and then import the XML file in FCPX into Da Vinci, and find the materials that need to be linked in the media pool, so that the FCPX timeline can be presented in Da Vinci, and the toning.
> 
> And the reason why I was excited was because I found the function of "loading CSV data" in CommandPost, so I tried to import all the materials in DaVinci first, and directly export a metadata file with the suffix "CSV", Then I put this file through the "Load CSV data" function, hoping to replace the starting timecode of the material in FCPX. It may be a mistake. I am very sure that I tried it for the first time, it was successful, and I Exporting XML files from FCPX can be opened directly by DaVinci without any errors in the middle, and even checked through DaVinci's offline reference fragments, there are no errors.
> 
> For the first time, there may be few materials, and the materials are all in one folder. For large-scale projects and projects with material management specifications, the reference is not meaningful. It can only be said to be a nice try. So I tried to test with a normal smooth and normal amount of material. I found that it seems that the time code of the material that has been renamed by me can be modified, but the time code of the material that has not been renamed cannot be modified.
> 
> So I made a third attempt, this time I only had three folders, two of which were SNOY footage, one was DJI aerial footage, and there was a file containing SNOY footage. All clips in the folder are renamed. The remaining two folders were not renamed.
> 
> This time... none of them were successful. I tried for 3 hours, and none of them were successful. This time, whether it was renaming the material or the material that was not renamed. I even tried, renaming all the footage, without success. I'm hoping for some help, am I doing something wrong with how I'm using this feature, or what I'm overlooking.
> 
> I hope you can help me with my doubts, thanks

**Reply by latenitefilms** [reply]
Unfortunately I don't think this "Toolbox" is what you want. It's really only useful for manipulating titles text.

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/22286696/181471347-bc0bcd51-afc9-478a-b793-ed4b48f69092.png">
